openwisp_radius/tests/test_tasks.py

```diff
@@ -173,6 +174,10 @@ def test_send_login_email(self, translation_activate, logger):
                 ' and/or traffic has been used or you can terminate the session',
                 ' '.join(email.alternatives[0][0].split()),
             )
+            self.assertIn(
+                'NOTE: This link is valid only for 2 hours',
+                ' '.join(email.alternatives[0][0].split()),
+            )
             translation_activate.assert_called_with(user.language)
```

Comment:
We should add a separate subtest for sesame token. In the current subtest, we should check that the expiration text is not present if `SESAME_MAX_AGE` variable is not set.